www/js/overlay.js

```diff
@@ -31,14 +33,20 @@ const savePrefs = () => {
   ipcRenderer.send('update-overlay-vars');
 };
 
-const colorInputFactory = (inputName, label, defaultColor, onchangeAction) =>
+const colorInputFactory = (inputName, defaultColor, onchangeAction) =>
   h(
     `div.${inputName}.input-wrap`,
     h(`input.${inputName}.color-input`, {
       type: 'color',
       onchange: onchangeAction,
       value: defaultColor,
     }),
+  );
+
+const controlsFactory = (controls, label) =>
+  h(
+    'div.toggle-text',
+    Array.isArray(controls) ? controls.map(c => c) : controls,
```

Comment:
? is this to make a copy of the array? I don't see the point of mapping if `controls` is already an array.